properties-files/openshiftv4_on_baremetal.properties

```diff
@@ -0,0 +1,46 @@
+ORCHESTRATION_USER=kni
+ORCHESTRATION_HOST=
+REBUILD_ORCHESTRATION=
+SSHKEY_TOKEN=
+SCALE_CI_BUILD_TRIGGER=true
+SCALE_CI_BUILD_TRIGGER_URL=
+ENABLE_DITTYBOPPER=
+KUBECONFIG_PATH=/home/kni/clusterconfigs/auth/kubeconfig
+OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_IMAGE_REGISTRY=
+OPENSHIFT_INSTALL_REGISTRY_TOKEN=
+CLOUD_NAME=
+LAB_NAME=
+ROUTABLE_API=
+JUMBO_MTU=
+BAREMETAL_NETWORK_CIDR=
+BAREMETAL_NETWORK_VLAN=
+ANSIBLE_SSH_PASSWORD=
+OPENSHIFT_BASE_DOMAIN=
+OPENSHIFT_CLUSTER_NAME=
+OPENSHIFT_RELEASE=
+OPENSHIFT_BUILD=
+OPENSHIFT_INSTALL_PULL_SECRET=
+OPENSHIFT_WORKER_COUNT=2
+OPENSHIFT_POST_CONFIG=
```

Comment:
The post-config steps which includes the performance dashboards and cerberus setup are part of https://github.com/cloud-bulldozer/scale-ci-deploy/tree/master/OCP-4.X/roles/post-install. We might want to add a step to consume those. For example: https://github.com/cloud-bulldozer/scale-ci-pipeline/blob/master/jjb/dynamic/scale-ci_install_osp.yml#L51.

Reply:
Yes first let the install job go through. I'm mostly interested in that. We can build on that later.